reference/catalog.v3.yml

```diff
@@ -21930,13 +21874,56 @@ components:
           x-required:
             - post
         amount:
-          type: integer
+          oneOf:
+            - type: number
+              example: 10
+            - type: string
+              example: '.10'
           description: |-
-            The discount can be a fixed dollar amount or a percentage. For a fixed dollar amount enter it as an integer and the response will return as an integer. For percentage enter the amount as the percentage divided by 100 using string format. For example 10% percent would be “.10”. The response will return as an integer.
+            You can express the discount as either a fixed dollar amount or a percentage. To use a fixed dollar amount, send a number; the response will return a number. To use a percentage, divide the percentage by 100 and send the result in string format. For example, represent 10% as “.10”. The response will return a float.
```

Comment:
we have 3 types of discount amount, and it's not clear to me whether the fixed dollar amount expresses price discounts or fixed discounts.